### ISSUE TITLE
Handle FD B2 confirmation message

### ISSUE THREAD

**Issue by ngardiner** [ngardiner/TWCManager#107]
Sending the Stop command (which triggers a hard stop via the contactor being de-energised) results in a confirmation message being sent by Slave TWCs in the following format:

```UNKNOWN MESSAGE FROM SLAVE:FD B2 NN NN 00 00 00 00 00 00 00 00 00 00 00 5A```

NN NN denotes the Slave TWC's TWC ID. We should trap this message within the protocol parser to avoid reporting an unknown message when the Stop command is used.

**Comment by ngardiner** [comment]
This is now trapped as of commit 0828301 and no longer generates an unknown message warning. We also have the following to trap for Start Charging command:

```
*** UNKNOWN MESSAGE FROM SLAVE:FD B1 58 50 00 00 00 00 00 00 00 00 00 00 00 59
```

**Comment by ngardiner** [comment]
This is fixed in commit 1396c41.
